.github/workflows/test.yml

```diff
@@ -25,22 +25,18 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
+        # maximize CI coverage of different platforms and python versions while minimizing the
+        # total number of jobs. We run all pytest splits with the oldest supported python
+        # version (currently 3.9) on windows (seems most likely to surface errors) and with
+        # newest version (currently 3.12) on ubuntu (to get complete coverage on unix). We
+        # ignore mac-os, which is assumed to be similar to ubuntu.
```

Comment:
I think this should be removed now that it's testing `macos-latest` now?

Reply:
good catch!